python/ray/serve/_private/client.py

```diff
@@ -135,7 +135,15 @@ def _wait_for_deployments_shutdown(self, timeout_s: int = 60):
         """
         start = time.time()
         while time.time() - start < timeout_s:
-            deployment_statuses = self.get_serve_status().deployment_statuses
+            statuses_bytes = ray.get(
+                self._controller.get_all_deployment_statuses.remote()
+            )
+            deployment_statuses = [
```

Comment:
Will the deployment names in `deployment_statuses` include their app names? Is there some way we can get the app name here? Otherwise, [the `TimeoutError` later on](https://github.com/ray-project/ray/blob/f1b1e3992348f11a6e69f7cd91bbeaa9ebe83637/python/ray/serve/_private/client.py#L161) could be confusing if there's deployments with the same name from multiple apps.

Reply:
Yes, the deployment names will include their app names here.